### ISSUE TITLE
a version of `Input.is_key_pressed()` that uses `physical_keycode`

### ISSUE THREAD

**Issue by JapanYoshi** [godotengine/godot#56211]
### Godot version

3.4.stable.mono.official

### System information

Windows 10, GLES2, Vulkan

### Issue description

I want to check if a physical key (layout-agnostic) is currently pressed. For example, I want the player to use a different attack when the character is looking upwards, as opposed to sideways. To check if the player should do the Up Attack, I want to check if the W key is pressed on QWERTY, but the Z key on AZERTY.

However, I can't check if a key is pressed using `physical_keycode`, but only the platform/layout-dependent `keycode`.

I can think of a workaround using the InputMap, but I have a total of 8 keys that I want to check this way, so it's inconvenient and impractical.

### Steps to reproduce

1. Set up an input event where it checks for a key (in this example, the W key). For example:

```
func _input(event):
    if event is InputEventKey:
        var sc = event.physical_scancode
        if sc == KEY_SPACE and event.pressed:
            if Input.is_key_pressed(KEY_W):
                print("Up attack")
            else:
                print("Normal attack")
        elif sc == KEY_W: # W key on QWERTY, types comma in Dvorak
            if event.pressed:
                print("W pressed")
            else:
                print("W released")
        elif sc == KEY_COMMA: # Comma key on QWERTY, types W in Dvorak
#        elif sc == KEY_Z # Use this instead if testing on AZERTY
            if event.pressed:
                print("comma pressed")
            else:
                print("comma released")
```

2. Play the game with an alternative keyboard layout enabled that remaps the W key, e.g. Dvorak,

Expected result, when holding W, pressing Space, releasing W, pressing Space, holding Comma, pressing Space, releasing Comma, then pressing Space
```
W pressed
Up attack
W released
Normal attack
comma pressed
Normal attack
comma released
Normal attack
```

Actual result.
```
W pressed
Normal attack
W released
Normal attack
comma pressed
Up attack
comma released
Normal attack
```

**Comment by Chaosus** [comment]
Note: this was implemented in 4.0 and Input has the `is_physical_key_pressed` method in that branch. Doing so for 3.x may require a big amount of work to implement that system.

**Comment by Calinou** [comment]
https://github.com/godotengine/godot/pull/55251 was cherry-picked for 3.5.
